### ISSUE TITLE
Ensure ink text is short and concise

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#60]
## Description
In describing the target of the anchor, anchor text should be short and to the point. Long link text is often cumbersome for users who are non-visual and generally can be refined to be more effective.

### SlideBlock

- [x] Link text is entire slide contents but should probably just be the title of the slide.  Mmost straight forward fix is probably to make only the title a link instead of the whole slide.   But I think it's reasonable to fix using `aria-label` or `aria-labeledby` attributes, or by leaving as is and adding `aria-hidden="true"` to all elements inside the link except the title.  Defer to y'all's best judgement.

### News Grid

- [x] Link text is entire card contents but should probably just be the title of the slide.  Most straight forward fix is probably to make only the title a link instead of the whole card.   But I think it's reasonable to fix using `aria-label` or `aria-labeledby` attributes, or by leaving as is and adding `aria-hidden="true"` to all elements inside the link except the title.  Defer to y'all's best judgement.

![8ed77cfda8c7fe6613e2b1ea59727218eb13acd2-1641195381](https://user-images.githubusercontent.com/8166060/152620063-000a9a7c-1756-4ea2-aaf5-d8a90a636fb5.png)

**Comment by dananjohnson** [comment]
@blnkt for `SlideBlock` here, I'd lean towards using `aria-labelledby` on the `a` tag rather than using `aria-hidden` for the link contents outside of the title. That way if someone is using a screen reader in reading mode, they would still get a read-out of the contents of the slide. What do you think?

**Comment by blnkt** [comment]
Using `aria-labelledby` sounds like a good plan to me
